roles/pulp-api/tasks/sso-configuration.yml

```diff
@@ -179,6 +187,12 @@
   when: keycloak_host_loopback_available | default(False)
   no_log: true
 
+- name: Store galaxy_verify_keycloack_ssl_certs
+  set_fact:
+    galaxy_verify_keycloack_ssl_certs: "{{ _sso_configuration.resources[0].data['galaxy_verify_keycloack_ssl_certs_available'] | b64decode }}"
```

Comment:
this should be `_sso_configuration.resources[0].data['galaxy_verify_keycloack_ssl_certs']`